tests/trace/util.py

```diff
@@ -5,7 +5,10 @@
 
 
 def client_is_sqlite(client):
-    return isinstance(client.server._internal_trace_server, SqliteTraceServer)
+    return isinstance(
+        client.server._next_trace_server.server._internal_trace_server,
```

Comment:
This change is because we have a new middleware layer. this check will need to change anyway in the near future